src/main/java/org/opensrp/web/rest/EventResource.java

```diff
@@ -399,7 +400,7 @@ private void searchMissingClients(List<String> clientIds, List<Client> clients,
 
 	@Override
 	public Event create(Event o) {
-		return eventService.addEvent(o);
+		return eventService.addEvent(o, ""); //TODO?

```

Comment:
Implement this TODO. You can change so that this does not override the create. The default create can just return a not-supported exception

You can also tag `@RequestMapping(method = RequestMethod.POST, consumes = { MediaType.APPLICATION_JSON_VALUE }, produces = { MediaType.APPLICATION_JSON_VALUE })` on the method.

Reply:
Updated.